app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ReviewsController < ApplicationController
+  def new
+    # unless
```

Comment:
If this is commented out code, remove. If this is explaining the code, it could be clearer. 🍨